README.md

```diff
@@ -1,2 +1,59 @@
 # beprof
 Beam Profile Analysing Tools
+
+Library provides methods to work with Beam Profiles which are sets of points <br>
+(might be 2-D or 3-D also with extra metadata) sorted by one of coordinates.
+
+beprof is based on nparray class from numpy, and it provides <br>
+numerous tools for different computations and data analysis.
+
+# installation
+
+Current version (0.1.0) is not available on PyPi, although once a <br>
+stable version is ready it will be pushed to PyPi repo.<br><br>
+
+For now, installation can be done from this GIT repository, using <br>
+`~$ pip3 install git+https://github.com/grzanka/beprof.git@master` <br>
+
+(where `@master` refers to the name of a branch)
+
+To unistall, simply use:<br>
+
+`~$ pip3 uninstall beprof`<br>
+
+
+# usage
+
+Once you install beprof, you should be able to import is as a python module<br>
+Using ipython the code would be i.e.<br>
+```
+>>> import beprof
+>>> from beprof import curve  #imports curve module
+>>> from beprof import profile  #imports profile module
```

Comment:
When you add the ">>>" at the beginning of the line I get angry when trying to copy-paste. I select block of 3 lines, paste it into python interpreter and get this:

```
Leszeks-MacBook-Air:rbe-o-meter grzanka$ python3
Python 3.5.1 (v3.5.1:37a07cee5969, Dec  5 2015, 21:12:44) 
[GCC 4.2.1 (Apple Inc. build 5666) (dot 3)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> >>> import beprof
  File "<stdin>", line 1
    >>> import beprof
     ^
SyntaxError: invalid syntax
>>> >>> from beprof import curve  #imports curve module
  File "<stdin>", line 1
    >>> from beprof import curve  #imports curve module
     ^
SyntaxError: invalid syntax
>>> >>> from beprof import profile  #imports profile module
  File "<stdin>", line 1
    >>> from beprof import profile  #imports profile module
     ^
SyntaxError: invalid syntax
```

Its annoying to realize that you need to copy line-by-line.